roxctl/deployment/check/check.go

```diff
@@ -35,6 +38,7 @@ const (
 )
 
 var (
+	log = logging.CreateLogger(logging.CurrentModule(), 0)
```

Comment:
> The environment has a logger attached to it that you should use: d.env.Logger().InfofLn

Oh even better, I'll use that one then :+1:

Reply:
Using `d.env.Logger().InfofLn` has the unintended side-effect of making most tests fail, since they assert against what messages are logged. I'll need to either add that line to the tests or remove the log after all.

Reply:
Changing the level of the log message you added to `DEBUG` likely would solve this as well.

Reply:
There is no debug log level in the logger we use there, and now we have agreed on just removing the log line.

Reply:
Let's remove this here then?